### ISSUE TITLE
得到的几乎全是黑图,也去不了背景，怎么回事？

### ISSUE THREAD

**Issue by gg22mm** [PaddlePaddle/PaddleSeg#2312]
<p>https://github.com/PaddlePaddle/PaddleSeg/blob/release/2.5/Matting/README_CN.md<br />
  训练后 运行得到的模型   全是黑图 怎么回事<br />
  <br />
  python predict.py --config ./configs/modnet/modnet_mobilenetv2.yml --model_path ./output/best_model/model.pdparams --image_path ./data/PPM-100/val/fg/ --save_dir ./output/results <br />
  <br />
  <img src="http://ztg.diyyq.com/123/1657592863611.png" /><br />
  <img src="http://ztg.diyyq.com/123/1657592832154.png" /><br />
  <img src="http://ztg.diyyq.com/123/1657592893641.png" /></p>
<p>========================= end ========================<br />
  还在训练中.. <br />
  <img src="http://ztg.diyyq.com/123/end.png" /> </p>

==================== modnet_mobilenetv2.yml ====================

batch_size: 16
iters: 100000

train_dataset:
  type: MattingDataset
  dataset_root: data/PPM-100
  train_file: train.txt
  transforms:
    - type: LoadImages
    - type: ResizeByShort
      short_size: 512
    - type: RandomCrop
      crop_size: [512, 512]
    - type: RandomDistort
    - type: RandomBlur
    - type: RandomHorizontalFlip
    - type: Normalize
  mode: train

val_dataset:
  type: MattingDataset
  dataset_root: data/PPM-100
  val_file: val.txt
  transforms:
    - type: LoadImages
    - type: ResizeByShort
      short_size: 512
    - type: ResizeToIntMult
      mult_int: 32
    - type: Normalize
  mode: val
  get_trimap: False

model:
  type: MODNet
  backbone:
    type: MobileNetV2
    pretrained: https://paddleseg.bj.bcebos.com/matting/models/MobileNetV2_pretrained/model.pdparams
  pretrained: Null

optimizer:
  type: sgd
  momentum: 0.9
  weight_decay: 4.0e-5

lr_scheduler:
  type: PiecewiseDecay
  boundaries: [40000, 80000]
  values: [0.02, 0.002, 0.0002]

**Comment by wuyefeilin** [comment]
训练梯度爆炸了。 另外PPM-100是用于教程演示的哈，数据量很小很难训练得到一个收敛的模型的。

如果你只是预测的话可以利用我们提供的训练好的模型进行预测哈

**Comment by gg22mm** [comment]
哦哦，这个演示能不能搞真实一点  ^_^!   万一是问题呢~~ ，
